lib/symbioticpy/symbiotic/options.py

```diff
@@ -200,6 +200,171 @@ def parse_command_line():
+                                  'is a string of type when-level, where when is \'before\'\n'
+                                  'or \'after\' (slicing) and level in \'conservative\', \'klee\'\n'
+                                  '\'O2\', \'O3\'. A special value is \'none\', which\n'
+                                  'disables optimizations (same as --no-optimize).\n'
+                                  'You can also pass optimizations directly to LLVM\'s opt,\n'
+                                  'by providing a string when-opt-what, e.g. before-opt-iconstprop'))
+    arg_parser.add_argument('--prp', dest='verified_property',
+                            choices=['null-deref', 'memsafety', 'undefined-behavior',
+                                     'undef-behavior', 'undefined', 'signed-overflow',
+                                     'no-overflow', 'termination', 'coverage'],
+                            help=('Specify property that should hold. It is either LTL formula\n'
+                                  ' as specivied by SV-COMP, or one of following shortcuts:\n'
+                                  ' null-deref         -- program is free of null-dereferences\n'
+                                  ' memsafety          -- program does not use invalid memory\n'
+                                  '                       (e.g., no double-free,\n'
+                                  '                       no invalid dereference, etc.)\n'
+                                  ' undefined-behavior -- check for undefined behaviour\n'
+                                  ' undef-behavior\n'
+                                  ' undefined\n'
+                                  ' signed-overflow -- check for signed integer overflow\n'
+                                  ' no-overflow\n'
+                                  ' termination     -- try checking whether the program\n'
+                                  '                    always terminates\n'
+                                  ' coverage        -- generate tests with as high coverage\n'
+                                  '                   as possible'))
+
+    arg_parser.add_argument('--verifier', metavar='VERIFIER_TOOL',
+                            choices=['Ceagle', 'KLEE', 'CPAchecker', 'Seahorn', 'Skink', 'Smack'],
+                            help=('Use the tool \'name\'. Default is KLEE, other tools that\n'
+                                  'can be integrated are Ceagle, CPAchecker, Seahorn,\n'
+                                  'Skink and SMACK.'))
+
+    arg_parser.add_argument('--target', metavar='VERIFIER_TOOL',
+                            choices=['Ceagle', 'KLEE', 'CPAchecker', 'Seahorn', 'Skink', 'Smack'],
+                            help=('Use the tool \'name\'. Default is KLEE, other tools that\n'
+                                  'can be integrated are Ceagle, CPAchecker, Seahorn,\n'
+                                  'Skink and SMACK. Synonymous to --verifier.'))
```

Comment:
You can combine these options into one:
```suggestion
    arg_parser.add_argument('--target', '--verifier', default='klee', ...)
```
Also, since KLEE is the default, it should be reflected using the default kwarg. Don't forget to update the help message. 

I'm not that familiar with this part of Symbiotic, but the list of all supported verifiers should be stored in the following dictionary, so the current choices in the argument won't actually work (@mchalupa can you please confirm?):

https://github.com/staticafi/symbiotic/blob/main/lib/symbioticpy/symbiotic/targets/__init__.py#L24-L52